app/controllers/broker_agencies/applicants_controller.rb

```diff
@@ -48,7 +48,7 @@ def update
       flash[:notice] = "Applicant accepted successfully."
     end
 
-    redirect_to broker_agencies_profile_path(@broker_agency_profile)
+    # redirect_to broker_agencies_profile_path(@broker_agency_profile)
```

Comment:
If we are not going to redirect to this endpoint, it’s better to either delete it or update it to the correct endpoint.